### PR TITLE
SC: add unit tests for multi operators

### PR DESCRIPTION
**Pull request by jeonghonah**
## Proposed changes

- Unit tests for multi bridge/operator which is merged at #80 
- This test case is based on the following document: https://docs.google.com/spreadsheets/d/1aQZVWTNGs6KLHKXUPxHDX5akGUBiGDS9xAexLG6zMfE/edit?pli=1#gid=2142770539

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: node/sc/bridge_manager.go [hunk 1]
```diff
@@ -46,6 +46,11 @@ const (
 	ERC721
 )
 
+const (
+	voteTypeValueTransfer = 0
+	voteTypeConfiguration = 1
```
Comment (aidan-kwon) [inline node/sc/bridge_manager.go]: Why thises variables are not declared in `multi_bridge_test.go`, but `bridge_manager.go`?

Comment (jeonghonah) [reply node/sc/bridge_manager.go]: If we add api for some bridgeOperator contract methods later, the constants are necessary.

File: node/sc/multi_bridge_test.go [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+			// expected timeout
+			return
+		}
+	}
+	t.Fatal("fail to check event result")
+}
+
+// TestMultiBridgeKLAYTransfer3 checks the following:
+// - tx is actually handled when operator threshold is satisfied.
+// - No double spending is made due to additional tx.
```
Comment (jeonghonah) [inline node/sc/multi_bridge_test.go]:
```suggestion
// - no double spending is made due to additional tx.
```

File: node/sc/bridge_manager.go (hunk 1 above)
Comment (aidan-kwon) [reply node/sc/bridge_manager.go]: Okay. Thank you.

File: node/sc/multi_bridge_test.go [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+	assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
+loop:
+	for {
+		select {
+		case ev := <-info.handleCh:
+			assert.Equal(t, nonceOffset, ev.HandleNonce)
+			return
+		case err := <-info.handleSub.Err():
+			t.Log("Contract Event Loop Running Stop by sub.Err()", "err", err)
```
Comment (laggu) [inline node/sc/multi_bridge_test.go]: How about using assert.FailNow here?

Comment (jeonghonah) [reply node/sc/multi_bridge_test.go]: Fixed

File: node/sc/multi_bridge_test.go [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+	acc = info.accounts[1]
+	tx = SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
+	info.sim.Commit()
+	assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
+	for {
+		select {
+		case _ = <-info.handleCh:
+			acc = info.accounts[2]
+			tx = SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
```
Comment (aidan-kwon) [inline node/sc/multi_bridge_test.go]: How can we sure this tx cannot trigger any bridgeHandle event?

Comment (jeonghonah) [reply node/sc/multi_bridge_test.go]: We check assert.Error()

Comment (aidan-kwon) [reply node/sc/multi_bridge_test.go]: Thank you!

File: node/sc/multi_bridge_test.go [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+
+	nonceOffset := uint64(17)
+	sentNonce := nonceOffset
+	transferAmount := uint64(100)
+	sentBlockNumber := uint64(100000)
+
+	tx = SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
+	info.sim.Commit()
+	assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
```
Comment (KimKyungup) [inline node/sc/multi_bridge_test.go]: If possible, we need to check the vote count and if it is executed or not in next PR.

File: node/sc/multi_bridge_test.go [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+	for {
+		select {
+		case _ = <-info.handleCh:
+			t.Fatal("unexpected handling of value transfer")
+		case err := <-info.handleSub.Err():
+			t.Fatal("Contract Event Loop Running Stop by sub.Err()", "err", err)
+		case <-time.After(timeOut):
```
Comment (KimKyungup) [inline node/sc/multi_bridge_test.go]: If possible to check the vote count in next time, it is better. :)

**Pull request merged**